src/SprykerEco/Zed/Unzer/Business/Import/UnzerPaymentMethodImporter.php

```diff
@@ -88,4 +108,54 @@ public function performPaymentMethodsImport(UnzerKeypairTransfer $unzerKeypairTr
 
         $this->paymentImportAdapter->createPaymentMethods($paymentProviderTransfer, $paymentMethodTransfers);
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\PaymentMethodsTransfer $paymentMethodsTransfer
+     * @param \Generated\Shared\Transfer\PaymentMethodsTransfer $childPaymentMethodsTransfer
+     *
+     * @return \Generated\Shared\Transfer\PaymentMethodsTransfer
+     */
+    protected function appendChildPaymentMethodsTransfer(
```

Comment:
```suggestion
    protected function appendChildPaymentMethods(
```

since it confusing now
it appends singular `PaymentMethodTransfer` object one by one

Reply:
Fixed.